Apps/Src/Tasks.c

```diff
@@ -54,12 +54,13 @@ extern const pinInfo_t PININFO_LUT[]; // For GPIO writes. Externed from Minions
  * Error assertion-related functions
 */
 
-void _assertOSError(OS_ERR err)
+void _assertOSError(volatile OS_ERR err)
```

Comment:
We made this volatile when we were testing a specific OS fault related bug. At a certain section in ReadTritium, there was an extra assertOSError left in that at certain points was being called on an uninitialized OS_ERR variable. This means it was essentially calling assertOSError on garbage data (since uninitialized variables become whatever used to be on the stack). To solve this issue, we printed out the error value in this function through GDB first, and to do so, I had to make this volatile so that GCC didn't optimize the value out and so we could see it in GDB.

However, now that debugging is done, I'll remove this volatile as well as the print statement below.